pkgs/tools/security/swtpm/default.nix

```diff
@@ -21,17 +24,17 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-7YzdwGAGECj7PhaCOf/dLSILPXqtbylCkN79vuFBw5Y=";
   };
 
-  patches = [
```

Comment:
you should keep the patch to allow doCheck = false; builds. For example, doCheck is set to false in crossCompilation.

Reply:
See previous comment.